packages/components/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 <!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/HEAD/packages#maintaining-changelogs. -->
 
 ## Unreleased
+-   `BorderControl`: Apply proper metrics and simpler text ([#53998](https://github.com/WordPress/gutenberg/pull/53998)).
```

Comment:
CHANGELOG entries should all go under a sub-section — ie. `Enhancements`, `Bug fix`, etc

We can tweak this in following PRs (eg. #54394)